composer.json

```diff
@@ -30,10 +30,20 @@
     },
     "autoload": {
         "psr-4": {
-            "SilverStripe\\Forms\\": "code/",
-            "SilverStripe\\Forms\\Tests\\": "tests/"
+            "SilverStripe\\Forms\\": "src/"
+        }
+    },
+    "autoload-dev": {
```

Comment:
`autoload-dev` will cause problems when running tests locally if the module was installed into the vendor directory of an existing project (which is how the CMS Squad works most of the time).
It also prevents us from running these tests in conjunction with other modules e.g. as part of recipe-kitchen-sink.

Please revert this change.